### ISSUE TITLE
could not export rules in v6r2013

### ISSUE THREAD

**Issue by GoogleCodeExporter** [moxter/mxupdate-archive#164]
```
What steps will reproduce the problem?
1. try to execute export of a rule (for example rule "SourcingBuyerRule") see 
command from attached file
2.
3.

What is the expected output? What do you see instead?
pls. see attached file v6r2013_rule_export.txt

What version of the product are you using? On what operating system?
V6R2013 beta 1210 on Windows Server 2008 R2

Please provide any additional information below.

```

Original issue reported on code.google.com by `Jens.Sch...@de.soliver.com` on 13 Mar 2012 at 2:41

Attachments:
- [v6r2013_rule_export.txt](https://storage.googleapis.com/google-code-attachments/mxupdate/issue-164/comment-0/v6r2013_rule_export.txt)

**Comment by GoogleCodeExporter** [comment]
```
Same applies to policies (with url 
/stateDefList/stateDef/userAccessList/userAccess/userAccessKind), so the main 
issue seems to be the "userAccessKind" definition. I could provide example 
output for policies too if needed.
```

Original comment by `Jens.Sch...@de.soliver.com` on 13 Mar 2012 at 2:49

**Comment by GoogleCodeExporter** [comment]
```
This may be related to issue 163.
```

Original comment by `Jens.Sch...@de.soliver.com` on 15 Mar 2012 at 8:33

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Please find unofficial fix from issue #163.
```

Original comment by `mx.syste...@gmail.com` on 9 May 2012 at 4:28

**Comment by GoogleCodeExporter** [comment]
Original comment by `tim.mox...@innobix.com` on 24 Sep 2013 at 9:06
- Added labels: **R2013x**, **V6**

**Comment by GoogleCodeExporter** [comment]
Original comment by `tim.mox...@innobix.com` on 24 Sep 2013 at 9:22
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
Original comment by `tim.mox...@innobix.com` on 24 Sep 2013 at 9:22

**Comment by GoogleCodeExporter** [comment]
Original comment by `tim.mox...@innobix.com` on 6 Oct 2013 at 2:06
- Changed state: **Duplicate**

**Comment by GoogleCodeExporter** [comment]
```
merged into issue #177, issue #180 and issue #181
```

Original comment by `tim.mox...@innobix.com` on 6 Oct 2013 at 2:07

**Comment by GoogleCodeExporter** [comment]
Original comment by `tim.mox...@innobix.com` on 7 Feb 2014 at 8:31
- Added labels: **MigrationStatus-Duplicate**
